### ISSUE TITLE
Tabs component: content gets cutoff at narrow viewports (or zoomed)

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#789]
<img width="415" alt="Screen Shot 2022-09-20 at 2 01 10 PM" src="https://user-images.githubusercontent.com/3180185/191363718-b8356368-40d4-4731-8fda-c4e19755f8cc.png">

TBD: what _should_ happen at narrow viewports for the tabs?

**Comment by tbusillo** [comment]
Ideally, I think the tabs would narrow to fit the viewport, but should shift to being vertically aligned if they cannot all fit. I don’t think just wrapping them if they overflow would look great.

Another option would be to allow for horizontal scrolling of the tabs if they overflow the container’s width. However, that approach may cause other accessibility issues.
